src/lib.rs

```diff
@@ -2,17 +2,17 @@ use std::time::{Duration, Instant};
 
 pub struct TokenBucket {
     capacity: usize,
```

Comment:
@malor Do you think `capacity` is a good name here, or shall we use `limit` instead?

Reply:
I used the name `limit` when I prototyped the algorithm implementation locally. I don't have a strong preference, though.

Reply:
I'll make a patch.